docs/design/agent-systemd-cgroup.md

```diff
@@ -0,0 +1,71 @@
+# Systemd Cgroup for Agent
+
+As we know, we can interact with cgroups in two ways, **cgroupfs** and **systemd**. The former is achieved by reading and writing cgroup tmpfs' files under `/sys/fs/cgroup` while the latter is done by configuring a transient unit by requesting systemd. Kata agent uses **cgroupfs** by default, unless you pass the parameter `--systemd-cgroup`.
+
+## usage
+
+For systemd, kata agent configures cgroups according to the following `linux.cgroupsPath` format standard provided by runc (`[slice]:[prefix]:[name]`). If you don't provide a valid `linux.cgroupsPath`, kata agent will treat it as `"system.slice:kata_agent:<container-id>"`. 
+
+> Here slice is a systemd slice under which the container is placed. If empty, it defaults to system.slice, except when cgroup v2 is used and rootless container is created, in which case it defaults to user.slice.
+>
+> Note that slice can contain dashes to denote a sub-slice (e.g. user-1000.slice is a correct notation, meaning a subslice of user.slice), but it must not contain slashes (e.g. user.slice/user-1000.slice is invalid).
+>
+> A slice of - represents a root slice.
```

Comment:
"A slice of - represents a root slice." -> "A slice of `-` represents a root slice."